tests/unit/nupic/algorithms/anomaly_test.py

```diff
@@ -115,5 +116,21 @@ def testComputeAnomalySelectModePure(self):
 
 
 
+  def testSerialization(self):
+    """serialization using pickle"""
+    # instances to test
+    aDef = Anomaly()
+    aLike = anomaly.Anomaly(mode=Anomaly.MODE_LIKELIHOOD)
+    aWeig = Anomaly(mode=Anomaly.MODE_WEIGHTED)
+#    aCust = Anomaly(mode=Anomaly.MODE_CUSTOM, customComputeFn=sum)
```

Comment:
Better not to leave dead code in PR. An exception might be if accompanied by a comment describing why it's left in.